.travis.yml

```diff
@@ -24,35 +24,38 @@ env:
 matrix:
   include:
     - php: 7.1
-      env: DB=PGSQL PHPUNIT_TEST=1 PHPCS_TEST=1
+      env: DB=MYSQL PHPUNIT_TEST=1 COMPOSER_ARG=--prefer-lowest
+    - php: 7.1
+      env: DB=PGSQL PHPUNIT_TEST=1
     - php: 7.2
       env: DB=MYSQL PHPUNIT_TEST=1
     - php: 7.3
       env: DB=MYSQL PDO=1 PHPUNIT_COVERAGE_TEST=1
-    - php: 7.4
-      env: DB=MYSQL PHPUNIT_TEST=1
-    - php: 7.1
-      env: DB=MYSQL PHPUNIT_TEST=1
-    - php: 7.2
-      env: DB=MYSQL NPM_TEST=1
     - php: 7.3
       env: DB=MYSQL BEHAT_TEST=1
+    - php: 7.4
+      env: DB=MYSQL PHPUNIT_TEST=1 PHPCS_TEST=1
+    - php: 7.4
+      env: DB=MYSQL NPM_TEST=1
+    - php: nightly
+      env: DB=MYSQL PHPUNIT_TEST=1 COMPOSER_ARG=--ignore-platform-reqs WITH_XDEBUG=1
+
 
 before_script:
   # Extra $PATH
   - export PATH=/usr/lib/chromium-browser/:$PATH
 
   # Init PHP
   - phpenv rehash
-  - phpenv config-rm xdebug.ini
+  - if [[ ! $WITH_XDEBUG ]] ; then phpenv config-rm xdebug.ini; fi
```

Comment:
Similar to one of the other PRs you opened - can you clarify the need for xdebug here?

Reply:
Discussion is in [silverstripe-reports PR](https://github.com/silverstripe/silverstripe-reports/pull/131#discussion_r502023196).